src/Roave/SecurityAdvisories/Advisory.php

```diff
@@ -45,6 +45,7 @@ final class Advisory
      */
     private function __construct(string $componentName, array $branchConstraints)
     {
+        /** @psalm-var callable(...VersionConstraint): array<VersionConstraint>|null $checkType */
```

Comment:
This `callable` should probably return a `list<VersionConstraint>`, no?

Reply:
Yep. Fixed